Services/Helpers/StorageMutex.cs

```diff
@@ -40,6 +40,11 @@ public StorageMutex(IStorageAdapterClient storageClient)
                     {
                         if (model.Metadata.ContainsKey(LAST_MODIFIED_KEY) && DateTimeOffset.TryParse(model.Metadata[LAST_MODIFIED_KEY], out var lastModified))
                         {
+                            // Timestamp retrieved successfully, nothing to do
```

Comment:
please add log here and line 47, so we can see what happens

Reply:
Fixed